README.md

```diff
@@ -107,6 +107,14 @@ $ fg
 ^C
 ```
 
+## Parameterized Queries 
+Must use HTTP POST with content-type=application/json. 'params' element must be an array in request body
+```console
+$ sqliteproxy --db currenttime.sqlite
+$ curl -i -X POST -H "Content-Type: application/json" -d "{\"sql\":\"select DATETIME(?) AS UTC_ISO\",\"params\":[\"now\"]}" http://localhost:2048
```

Comment:
1. I think `-i` is unnecessary here, as you're not showing the response headers one line below.
2. `-X POST` is unnecessary as well, as you're using `-d` which automatically sets `Request Method: POST`.
3. I think it would be nicer to use single-quotes to pass the JSON here, as you won't have to escape all the inner double-quotes: `-d '{"sql":"select DATETIME(?) AS UTC_ISO","params":["now"]}'`.

Let me know what you think, but IMO this looks cleaner:
```diff
- curl -i -X POST -H "Content-Type: application/json" -d "{\"sql\":\"select DATETIME(?) AS UTC_ISO\",\"params\":[\"now\"]}" http://localhost:2048
+ curl -H "Content-Type: application/json" -d '{"sql":"select DATETIME(?) AS UTC_ISO","params":["now"]}' http://localhost:2048
```

Reply:
your readme examples were from *nix. On windows, your curl commands wouldn't work (which is where I developed my PR). Feel free to replace my console example with your linux equivalent but please note that the linux command will not work on windows.